examples/traefik/registry-ui-nginx.conf

```diff
@@ -0,0 +1,48 @@
+      server {
+          listen       80;
+          server_name  localhost;
+          # disable any limits to avoid HTTP 413 for large image uploads
+          client_max_body_size 0;
+
+          # required to avoid HTTP 411: see Issue #1486 (https://github.com/moby/moby/issues/1486)
+          chunked_transfer_encoding on;
+          # required for strict SNI checking: see Issue #70 (https://github.com/Joxit/docker-registry-ui/issues/70)
+          proxy_ssl_server_name on;
+          proxy_buffering off;
+          # Fix push and pull of large images: see Issue #282 (https://github.com/Joxit/docker-registry-ui/issues/282)
+          proxy_request_buffering off; 
+          proxy_ignore_headers "X-Accel-Buffering";
+
+          location / {
+              root   /usr/share/nginx/html;
+              index  index.html index.htm;
+          }
+
+          location /v2 {
+              # Do not allow connections from docker 1.5 and earlier
+              # docker pre-1.6.0 did not properly set the user agent on ping, catch "Go *" user agents
+              if ($http_user_agent ~ "^(docker\/1\.(3|4|5(?!\.[0-9]-dev))|Go ).*$" ) {
+                  return 404;
+              }
+              proxy_ignore_client_abort on;
+              client_body_timeout 300;
+              proxy_read_timeout 300;
+              proxy_send_timeout 300;
+              keepalive_timeout 150;
+              proxy_http_version 1.1;
+              proxy_set_header Host "$http_host"; 
+              
+      #r        set $registry_server "http://registry:5000";
+      #r        proxy_pass $registry_server;
+              proxy_pass http://registry:5000;
```

Comment:
Why did you add the nginx configuration? It's not working with the bundled version?

Reply:
Only the following configuration is added to ensure that the default 60s is not interrupted when transferring large images.
```
              client_body_timeout 300;
              proxy_read_timeout 300;
              proxy_send_timeout 300;
              keepalive_timeout 150;
```

Reply:
Is this not necessary, or is it another issue?